docs/processing_information.md

```diff
@@ -175,3 +175,22 @@ The [decoy-aware reference transcriptome](https://salmon.readthedocs.io/en/lates
 
 A benefit of using `salmon` is the ability to incorporate RNA-seq specific technical biases and correct counts accordingly.
 We chose to enable the [`--seqBias`](https://salmon.readthedocs.io/en/latest/salmon.html#seqbias) and [`--gcBias`](https://salmon.readthedocs.io/en/latest/salmon.html#gcbias) flags, to correct for sequence-specific biases due to random hexamer primer and fragment-level GC biases, respectively.
+
+## Merged objects
+
+In addition to downloadable objects containing a single library, we also offer a merged object to download for each project, representing all libraries contained in a single file.
+This section describes how these merged objects were prepared.
+
+Merged objects were created from the processed `SingleCellExperiment` objects for each ScPCA project.
```

Comment:
I think instead of referring to the processed object here we should note at what point in processing the objects are used. This file doesn't explain much about what each of the objects contain, instead it talks about what steps were taken to create the data. 
```suggestion
Following post-processing of each `SingleCellExperiment` (see XXX section above on post-processing) object, all objects belonging to a single ScPCA project were merged together. 
```